docs/running-tests.rst

```diff
@@ -151,7 +151,7 @@ installation.
 .. note::
 
     It is not required to have the Active Directory machine running in order to
-    run the tests. If you run the tests with ``--mh-lazy-ssh`` (as shown in the
+    run the tests. If you run the tests with ``--mh-lazy-ssh -k "not ad"`` (as shown in the
```

Comment:
Can you please wrap it at 80 chars?

Reply:
Done.

Reply:
I think you forgot to push it :-)

Reply:
Fixed, this PR was not from my fork by accident.